data/presets/leisure/dancing_school.json

```diff
@@ -20,7 +20,10 @@
         "swing",
         "tango",
         "waltz",
-        "dance teaching"
+        "ballet",
+        "dance teaching",
+        "ballet studio",
+        "dance studio"
```

Comment:
Since there is an alias called "Dance Studio" here, does it make sense to add it as a term as well?

Reply:
No, it's not necessary to repeat the aliases here. Let's drop it here.

Reply:
Thanks for clarifying. Removed in [`6cf3fe4`](https://github.com/openstreetmap/id-tagging-schema/pull/1182/commits/6cf3fe4611f48a3ddcfee8e1e8c2b24a6fc7d0eb).